bundles/org.openhab.binding.exec/src/main/java/org/openhab/binding/exec/internal/handler/ExecHandler.java

```diff
@@ -292,7 +292,7 @@ public void execute() {
             String transformedResponse = StringUtils.chomp(outputBuilder.toString());
             String transformation = (String) getConfig().get(TRANSFORM);
 
-            if (transformation != null && transformation.length() > 0) {
+            if (transformation != null && transformation.length() > 0 && transformedResponse != null) {
```

Comment:
In line 292 you could do this to avoid this check:
```java
String transformedResponse = Objects.requireNonNull(StringUtils.chomp(outputBuilder.toString()));
```

It seems not possible for `chomp` to return null for `outputBuilder.toString()`.